python/paddle/incubate/nn/functional/fused_transformer.py

```diff
@@ -961,6 +970,84 @@ def fused_multi_head_attention(
         return (final_out, cache_kv_out) if cache_kv else final_out
 
 
+@overload
+def fused_multi_transformer(
+    x: Tensor,
+    ln_scales: Sequence[Tensor],
+    ln_biases: Sequence[Tensor],
+    qkv_weights: Sequence[Tensor],
+    qkv_biases: Sequence[Tensor],
+    linear_weights: Sequence[Tensor],
+    linear_biases: Sequence[Tensor],
+    ffn_ln_scales: Sequence[Tensor],
+    ffn_ln_biases: Sequence[Tensor],
+    ffn1_weights: Sequence[Tensor],
+    ffn1_biases: Sequence[Tensor],
+    ffn2_weights: Sequence[Tensor],
+    ffn2_biases: Sequence[Tensor],
+    pre_layer_norm: bool = True,
```

Comment:
```suggestion
    pre_layer_norm: bool = ...,
```

注意 `overload` 中默认值的写法